src/chapters/control-flow/loops.rst

```diff
@@ -0,0 +1,278 @@
+Loops
+=====
+
+.. index:: ! for loop
+
+``for`` Loop
+------------
+
+In Java we write a definite loop (aka a **for loop**) as:
+
+.. sourcecode:: java
```

Comment:
How about including a bash code block after this like we did in unit 1?  Preface the block with **Output**, then the code block showing 0 - 9.